global/easeprobe.go

```diff
@@ -43,7 +43,7 @@ func InitEaseProbe(name, icon string) {
 	easeProbe = &EaseProbe{
 		Name:    name,
 		IconURL: icon,
-		Version: Ver,
+		Version: VersionString(),
```

Comment:
I think it's better to have individual fields here. 

AppVersion - v1.6.0
GoVersion - v1.18.2
VCSReversion - afcdefg
VCSTime - 2022-....

Then, it can be easy to pick up the interests.  For example, we can get rid of the VCSTime in FootString...

Reply:
that way, it should call `runtime/debug` directly

I prefer not to add sha1 to footer

Reply:
either of below are too ugly/confused to me
EaseProbe v1.6.0 24a5f9109 2022-06-17T04:43:23Z go1.18.3 @ OMEN at 2022-06-17 08:57:39 UTC (timestamp ambiguity)
EaseProbe v1.6.0 24a5f9109 go1.18.3 @ OMEN at 2022-06-17 08:57:39 UTC (version ambiguity)
EaseProbe v1.6.0 24a5f9109 @ OMEN at 2022-06-17 08:57:39 UTC (why a sha1 here)

Reply:
Yes, it seems a bit messy. I think it's OK we keep the current notification footer with no change. What do you think?  @proditis

Reply:
Cool, let approve this PR